gateway-ha/pom.xml

```diff
@@ -137,18 +193,19 @@
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-servlets</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>jakarta.servlet</groupId>
+                    <artifactId>jakarta.servlet-api</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
 
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-views</artifactId>
         </dependency>
 
-        <dependency>
```

Comment:
We found that this is needed for the UI to work .. we might have to experiment with a different scope or somehow figure out how the dependency analyse run can be satisfied.

Reply:
See https://github.com/trinodb/trino-gateway/issues/213